src/main/java/it/gov/pagopa/merchant/controller/MerchantController.java

```diff
@@ -29,4 +31,9 @@ ResponseEntity<MerchantDetailDTO> getMerchantDetail(
             @PathVariable("initiativeId") String initiativeId,
             @PathVariable("merchantId") String merchantId
     );
+    @Operation(summary = "Returns the merchand id",
+            description = "")
+    @GetMapping("/{fiscalCode}/{acquirerId}/merchant-info")
```

Comment:
this path has not sense... If I should see just the URL, without seeing description or code, I would not be able to understand what I'm reading...

It's better something more verbose:
```suggestion
    @GetMapping("/acquirer/{acquirerId}/merchant-fiscalcode/{fiscalCode}/id")
```